.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: rust
 rust:
-- 1.22.0  # Two releases back
+- 1.16.0  # Min Rust Version
```

Comment:
Any particular reason you are targeting 1.16?

Reply:
`cargo check` 😆

Reply:
It's going to bump up again, I went back to 1.15 which is where custom derive was introduced and am pushing it up by the errors.

I think the latest thing I used is `?`, which I forgot about until just now.

Reply:
Basically, it's implemented now how I want it to be, and I'm trying to guarantee the largest range I can.